README.md

```diff
@@ -184,22 +249,39 @@ Message: Dimension 'CV mmHg*s虏/ml' not available in DimensionFactory.
-```
+-- On Windows, set
+`Settings > Language > Administrative language settings > Current language for non-Unicode programs`
+to `English (United States)` and reboot. -- On Linux, set the
+environment variable `LC_ALL` before starting R:
+
+```         
 export LC_ALL=en_US.UTF-8
 ```
 
-- **RStudio crashes when trying to load a workspace.** The ospsuite package uses the features implemented in PK-Sim and MoBi by creating `.NET` objects (e.g. a simulation) and using them from R. These objects cannot be saved as part of the workspace and reloaded on next start. When trying to do so, RStudio simply crashes. There is no possibility to overcome this limitation. To prevent RStudio from crashing, make sure to disable the check-box "Restore `.RData` into workspace at startup" in the options of RStudio. Keep in mind that you can also change this setting for specific projects.
-
-# Code of Conduct
-
-Everyone interacting in the Open Systems Pharmacology community (codebases, issue trackers, chat rooms, mailing lists etc.) is expected to follow the Open Systems Pharmacology [code of conduct](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODE_OF_CONDUCT.md).
-
-# Contribution
-
-We encourage contribution to the Open Systems Pharmacology community. Before getting started please read the [contribution guidelines](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CONTRIBUTING.md). If you are contributing code, please be familiar with the [coding standards](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODING_STANDARDS_R.md).
+-   **RStudio crashes when trying to load a workspace.** The ospsuite
+    package uses the features implemented in PK-Sim and MoBi by creating
+    `.NET` objects (e.g. a simulation) and using them from R. These
+    objects cannot be saved as part of the workspace and reloaded on
+    next start. When trying to do so, RStudio simply crashes. There is
+    no possibility to overcome this limitation. To prevent RStudio from
+    crashing, make sure to disable the check-box "Restore `.RData` into
+    workspace at startup" in the options of RStudio. Keep in mind that
+    you can also change this setting for specific projects.
+
+# Code of Conduct {#code-of-conduct}
+
+Everyone interacting in the Open Systems Pharmacology community
+(codebases, issue trackers, chat rooms, mailing lists etc.) is expected
+to follow the Open Systems Pharmacology [code of
+conduct](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODE_OF_CONDUCT.md).
+
+# Contribution {#contribution}
+
+We encourage contribution to the Open Systems Pharmacology community.
+Before getting started please read the [contribution
+guidelines](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CONTRIBUTING.md).
+If you are contributing code, please be familiar with the [coding
+standards](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODING_STANDARDS_R.md).
```

Comment:
Now that coding standards were moved to the developer-docs repo we should point directly there:
https://github.com/Open-Systems-Pharmacology/developer-docs/blob/main/ospsuite-r-specifics/CODING_STANDARDS_R.md